server/command.go

```diff
@@ -70,6 +70,58 @@ func (p *Plugin) handleHelp(args []string, extra *model.CommandArgs) (string, er
 	return p.getHelpText(), nil
 }
 
+func (p *Plugin) handleStartConfirm(args []string, extra *model.CommandArgs) (string, error) {
+	if len(args) > 1 {
+		return tooManyParametersText, nil
+	}
+	userID := extra.UserId
+	user, appErr := p.API.GetUser(userID)
+	if appErr != nil {
+		return "Cannot get user.", errors.Wrap(appErr, "cannot get user")
+	}
+	channelID := extra.ChannelId
+	channel, err := p.API.GetChannel(channelID)
+	if err != nil {
+		return "", err
+	}
+	message := ""
+	if channel.IsGroupOrDirect() {
```

Comment:
Shouldn't this be the other way around? Only show the warning when you are in a something that is not a group or direct channel?

Reply:
From what I understood about the issue was that users very often click the start meeting button so this is supposed to act as a way to confirm they want to start the meeting. Please let me know if otherwise

Reply:
The problem is about creating a meeting in a channel with lots of people. In general, creating a meeting in a direct message is a normal behaviour, but creating a meeting in a public channel might have been a misclick. Therefore we want to make sure that the warning appears when you try to create a meeting in public or private channels. For other cases (direct and group messages) it should create the meeting directly.

Nevertheless, I defer to @aaronrothschild to confirm what I just said, and any other detail needed.